.github/workflows/ci.yml

```diff
@@ -56,6 +56,15 @@ jobs:
         COVERAGE_FILE: .coverage.${{ runner.os }}-py${{ matrix.python }}
         CONTEXT: ${{ runner.os }}-py${{ matrix.python }}
 
+    - name: uninstall deps
+      run: pip uninstall -y tomlkit PyYAML python-dotenv
```

Comment:
`python-dotenv` is not an optional dep. no need to test for it

Reply:
Should I remove it from the test then ? It was previously set here:
```python
@pytest.mark.skipif(dotenv, reason='python-dotenv is installed')
def test_dotenv_not_installed(tmp_path):
    p = tmp_path / '.env'
    p.write_text('a=b')
```

Reply:
Yes, `python-dotenv` is no an optional dep and will be installed with `pydantic-settings`